nbdime/gitdiffdriver.py

```diff
@@ -21,21 +21,23 @@
 import io
 import os
 import sys
-from subprocess import check_call, check_output, CalledProcessError
+from subprocess import check_call, CalledProcessError
 
 from . import nbdiffapp
+from .args import add_git_config_subcommand
 from .utils import locate_gitattributes, ensure_dir_exists
 
-def enable(global_=False):
+
+def enable(scope=None):
     """Enable nbdime git diff driver"""
     cmd = ['git', 'config']
-    if global_:
-        cmd.append('--global')
+    if scope:
+        cmd.append('--%s' % scope)
 
     check_call(cmd + ['diff.jupyternotebook.command', 'git-nbdiffdriver diff'])
-    gitattributes = locate_gitattributes(global_)
+    gitattributes = locate_gitattributes(scope)
     if gitattributes is None:
-        print("No .git directory in %s, skipping git attributes" % path, file=sys.stderr)
+        print("No .git directory in %s, skipping git attributes" % gitattributes, file=sys.stderr)
```

Comment:
This message won't make sense. The arg should be the CWD, and the message should only appear for scope=None.